server/scsynth/scsynth_main.cpp

```diff
@@ -44,6 +44,38 @@ inline int setlinebuf(FILE* stream) { return setvbuf(stream, (char*)0, _IONBF, 0
 
 #endif
 
+#ifdef _WIN32
+// Add a warning when Windows Defender delays scsynth boot by 60+ seconds
+// cf. github issue #4368
+#    include <iostream>
+#    include <thread>
+#    include <mutex>
+#    include <chrono>
+bool bScsynthHasBooted = false;
+std::mutex bScsynthHasBootedMutex;
+std::condition_variable bScsynthHasBootedConditionVariable;
```

Comment:
a couple points here:
- these should all be marked `static`
- our [naming conventions](https://github.com/supercollider/supercollider/wiki/Cpp-code-style-guidelines#naming) say that globals should have a `g_` prefix to their names
- this `bFlagName` notation for variables is called [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) and it's something we typically avoid. going along with [webkit naming conventions](https://webkit.org/code-style-guidelines/#names-bool), `g_isScsynthBooted` would be more appropriate.
- the mutex and condition_variable relate more to the warning than to the flag, so maybe `g_scsynthBootWarningMutex` and `g_scsynthBootWarningConditionVariable` are more appropriate? just suggesting since you are going to have to edit these names anyway.